### PR TITLE
MBS-9919: Convert the place map tab to React

### PR DESCRIPTION
**Pull request by reosarevok**
This should be the last bit needed for https://tickets.metabrainz.org/browse/MBS-9919

### CODE REVIEW

File: root/place/PlaceMap.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import PlaceLayout from './PlaceLayout';
+
+type Props = {
+  +mapDataArgs: {
+    +draggable: boolean,
+    +place: {
+      +coordinates: CoordinatesT | null,
+      +name: string,
+    },
```
Comment (reosarevok) [inline root/place/PlaceMap.js]: Not too sure about this, not that it makes much of a difference what I put here in this case.

Comment (mwiencek) [reply root/place/PlaceMap.js]: Which bit aren't you sure about?

Comment (reosarevok) [reply root/place/PlaceMap.js]: Well, whether I've typed it right. Is \0 meant to be a boolean, or a number? (I'm assuming boolean).

Comment (mwiencek) [reply root/place/PlaceMap.js]: Yeah, `\0` and `\1` get encoded as booleans. It's mentioned here https://metacpan.org/pod/JSON#other-references though using `JSON::false` and `JSON::true` is clearer.

Comment (reosarevok) [reply root/place/PlaceMap.js]: Clearer as in "I might as well change to that while I'm at it"? :)

Comment (mwiencek) [reply root/place/PlaceMap.js]: Heh, it was just an observation, you can change it if you feel like but otherwise don't bother. :)

**Pull request merged**